js-library/src/request-verifiable-presentation.ts

```diff
@@ -28,7 +36,10 @@ export type CredentialsRequest = {
   jsonrpc: typeof JSON_RPC_VERSION;
   method: typeof VC_REQUEST_METHOD;
   params: {
-    issuer: IssuerData;
```

Comment:
The `issuerData` is now for the parameter only, while `CredentialsRequest` is the data sent with the window post message.